pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -0,0 +1,154 @@
+package nodenetworkstate
```

Comment:
Could you please remove the controller from the first PR? Keep only an empty manager, let's add types (API) and controller (this) later in separate PRs